README.md

```diff
@@ -1,8 +1,8 @@
 This Repository is A Bot that is written in the Sapphire FrameWork 
 
-Why Should you use it:
+# Why should you use OurTube?
 You should use Ourtube today for your every day life when working or Relaxing at home with your friends and Colleagues 
 
-Description of what the Bot does: 
+# Description
 you are able to play music from any platform  
```

Comment:
I thought we changed this to only playing copyright free music that's locally downloaded?